src/main/java/frc/team2412/robot/subsystem/IndexSubsystem.java

```diff
@@ -237,70 +230,66 @@ public boolean feederHasCorrectCargo() {
     // do need now! :D D: :3 8) B) :P C: xD :p :] E: :} :> .U.
     @Override
     public void periodic() {
-        ingestBallState = ingestSensorHasBallIn();
-        feederBallState = feederSensorHasBallIn();
-
-        // Checking for jamming
-        double ingestCurrent = ingestMotor.getSupplyCurrent();
-        if (ingestCurrent > CURRENT_LIMIT_TRIGGER_AMPS) {
-            if (ingestOverCurrentStart == 0) {
-                ingestOverCurrentStart = Timer.getFPGATimestamp();
-            }
-        }
-        if (ingestCurrent > CURRENT_LIMIT_RESET_AMPS) {
-            if (ingestOverCurrentStart > 0) {
-                if (Timer.getFPGATimestamp() - ingestOverCurrentStart > CURRENT_LIMIT_TRIGGER_SECONDS) {
-                    ingestMotorStop();
-                }
-
-            }
-        } else {
-            ingestOverCurrentStart = 0;
-        }
-
-        double feederCurrent = feederMotor.getSupplyCurrent();
-        if (feederCurrent > CURRENT_LIMIT_TRIGGER_AMPS) {
-            if (feederOverCurrentStart == 0) {
-                feederOverCurrentStart = Timer.getFPGATimestamp();
-            }
-        }
-        if (feederCurrent > CURRENT_LIMIT_RESET_AMPS) {
-            if (feederOverCurrentStart > 0) {
-                if (Timer.getFPGATimestamp() - feederOverCurrentStart > CURRENT_LIMIT_TRIGGER_SECONDS) {
-                    feederMotorStop();
-                }
-
-            }
-        } else {
-            feederOverCurrentStart = 0;
-        }
+        // ingestBallState = ingestSensorHasBallIn();
+        // feederBallState = feederSensorHasBallIn();
+        //
+        // // Checking for jamming
+        // double ingestCurrent = ingestMotor.getSupplyCurrent();
+        // if (ingestCurrent > CURRENT_LIMIT_TRIGGER_AMPS) {
+        // if (ingestOverCurrentStart == 0) {
+        // ingestOverCurrentStart = Timer.getFPGATimestamp();
+        // }
+        // }
+        // if (ingestCurrent > CURRENT_LIMIT_RESET_AMPS) {
+        // if (ingestOverCurrentStart > 0) {
+        // if (Timer.getFPGATimestamp() - ingestOverCurrentStart > CURRENT_LIMIT_TRIGGER_SECONDS) {
+        // ingestMotorStop();
+        // }
+        //
+        // }
```

Comment:
Just delete vs. commenting out, as it looks like `SupplyCurrentLimitConfiguration` handles the same thing (though would continue to let balls be further jammed vs. stopping the motors entirely).